assets/dev/js/editor/components/ai/pages/form-text/index.js

```diff
@@ -0,0 +1,177 @@
+	{ label: __( 'Make it shorter', 'elementor' ), icon: <ShrinkIcon />, value: 'Make the following message shorter' },
+	{ label: __( 'Fix spelling & grammar', 'elementor' ), icon: <AIIcon />, value: 'Fix the spelling and grammar of the following message' },
+];
+
+const promptInstructions = [
+	{ label: __( 'Change tone', 'elementor' ), options: vocalTones, getInstruction: ( value ) => `Change the tone of the following message to ${ value }` },
+	{ label: __( 'Translate to', 'elementor' ), options: translateLanguages, getInstruction: ( value ) => `Translate the following message to ${ value }` },
+];
+
+const FormText = (
+	{
+		type,
+		onClose,
+		getControlValue,
+		setControlValue,
+		additionalOptions,
+		credits: initialCredits,
+	},
+) => {
+	const initialValue = getControlValue() === additionalOptions?.defaultValue ? '' : getControlValue();
+	const {
+		prompt,
+		setPrompt,
+		send,
+		isLoading,
+		error,
+		resetError,
+		result,
+		setResult,
+		sendFeedback,
+		credits,
+	} = useTextPrompt( { initialResult: initialValue, initialCredits } );
+
+	const searchField = useRef( null );
+
+	const autocompleteItems = 'textarea' === type ? textareaAutocomplete : textAutocomplete;
+
+	const showSuggestions = ! prompt;
+
+	const handleSubmit = ( event ) => {
+		event.preventDefault();
+
+		send( prompt );
+	};
+
+	const handleCustomInstruction = async ( instruction ) => send( result, instruction );
+
+	const handleSuggestion = ( suggestion ) => {
+		setPrompt( suggestion + ' ' );
+		searchField.current.focus();
+	};
+
+	const applyPrompt = () => {
+		sendFeedback();
+
+		setControlValue( result );
+
+		onClose();
+	};
+
+	if ( isLoading ) {
+		return <Loader />;
+	}
+
+	return (
+		<>
+			{ error && <PromptErrorMessage error={ error } onClose={ resetError } sx={ { mb: 6 } } /> }
+
+			{ ! result && (
+				<Box component="form" onSubmit={ handleSubmit }>
+					<Box sx={ { mb: 6 } }>
+						<PromptSearch
+							ref={ searchField }
+							placeholder={ __( 'Describe the text and tone you want to use...', 'elementor' ) }
+							name="prompt"
+							value={ prompt }
+							onChange={ ( event ) => setPrompt( event.target.value ) }
+						/>
+					</Box>
+
+					{ showSuggestions && (
+						<PromptSuggestions
+							suggestions={ autocompleteItems }
+							onSelect={ handleSuggestion }
+							suggestionFilter={ ( suggestion ) => suggestion + '...' }
+						/>
+					) }
+
+					<Stack direction="row" alignItems="center" justifyContent="space-between" sx={ { py: 4, mt: 8 } }>
+						<PromptCredits credits={ credits } />
+
+						<GenerateButton>
+							{ __( 'Generate text', 'elementor' ) }
+						</GenerateButton>
+					</Stack>
+				</Box>
+			) }
+
+			{ result && (
+				<Box sx={ { mt: 3 } }>
+					<TextArea
+						value={ result }
+						onChange={ ( event ) => setResult( event.target.value ) }
+						helperText={ __( 'Text generated by A.I. may be inaccurate or offensive.', 'elementor' ) }
```

Comment:
sometimes you write "AI" and other times "A.I.", shouldn't be consistent?

Reply:
Done.